controllers/chat/getChannelDetail.js

```diff
@@ -4,7 +4,7 @@ const {User} = require('../../databases/models');
 const {ResponseSuccess} = require('../../utils/Responses');
 const UsersFunction = require('../../databases/functions/users');
 const BetterSocialCore = require('../../services/bettersocial');
-const {MESSAGE_TYPE} = require('../../helpers/constants');
+const {MESSAGE_TYPE, CHANNEL_TYPE} = require('../../helpers/constants');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CHANNEL_TYPE` has been added. This is necessary for the new logic introduced to determine whether or not to update better channel members based on the channel type.
```diff
- const {MESSAGE_TYPE} = require('../../helpers/constants');
+ const {MESSAGE_TYPE, CHANNEL_TYPE} = require('../../helpers/constants');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->